scripts/postinstall

```diff
@@ -2,25 +2,28 @@
 
 'use strict'
 
-const {reduce, forEach} = require('lodash')
 const jsonFuture = require('json-future')
-const getStdin = require('get-stdin')
+const got = require('got')
 
-;(async () => {
-  const buffer = await getStdin()
-  const {categories: services} = await jsonFuture.parseAsync(buffer)
+const DATASET_URL =
+  'https://rawgit.com/disconnectme/disconnect-tracking-protection/master/services.json'
 
-  const domains = Array.from(reduce(services, (set, service) => {
-    forEach(service, company => {
-      forEach(company, domains => {
-        forEach(domains, domain => {
-          set = new Set([...set, ...domain])
-        })
-      })
-    })
+const extractDomains = obj =>
+  Object.values(obj).reduce((domains, value) => {
+    if (Object.prototype.toString.call(value) === '[object Object]') return [...domains, ...extractDomains(value)]
+    if (Array.isArray(value)) return [...domains, ...value]
```

Comment:
If you put `if (Array.isArray(value))` statement first, then you can compare with `object` just using `typeof`, much clearer!